examples/crm/Cargo.toml

```diff
@@ -4,8 +4,11 @@ version = "0.1.0"
 authors = ["Denis Kolodin <deniskolodin@gmail.com>"]
 edition = "2018"
 
+[lib]
+crate-type = ["cdylib", "rlib"]
```

Comment:
We should if examples will be built as libraries or binaries. `wasm-bindgen` builds examples as binaries so I don't think this is needed